server/graphql/v1/mutations/expenses.js

```diff
@@ -212,6 +217,8 @@ export async function createExpense(remoteUser, expenseData) {
   const collective = await models.Collective.findByPk(expenseData.collective.id);
   if (!collective) {
     throw new errors.ValidationFailed('Collective not found');
+  } else if (![collectiveTypes.COLLECTIVE, collectiveTypes.EVENT].includes(collective.type)) {
+    throw new errors.ValidationFailed('Expenses can only be submitted to collectives and events');
```

Comment:
Hmm, why are we adding this limitation?
This seems to be an interesting feature.

Reply:
Expenses have always had this limitation, only before this PR it was only enforced in the frontend. The goal of this line is to make it explicit to prevent inconsistencies in the data if someone misuse the endpoint.